DocsAndHospitals.Infrastructure/Persistence/UserEfRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DocsAndHospitals.Models;
+using Microsoft.EntityFrameworkCore;
+using System.Threading.Tasks;
+
+namespace DocsAndHospitals.Persistence
+{
+    public class UserEfRepository : IUserRepository
```

Comment:
We don't name it Ef repo bc if we started with EF we 100% continue with EF. Never change the implementation in this case, then no need to name it with 'Ef'